plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/JdkLdapAuthenticatorClient.java

```diff
@@ -149,6 +155,31 @@ private CloseableContext createUserDirContext(String userDistinguishedName, Stri
         return environment.build();
     }
 
+    private static SSLContext createIgnoreCertificateSslContext()
+    {
+        try {
+            TrustManager[] trustManagers = {
+                new X509TrustManager() {
+                    @Override public void checkClientTrusted(X509Certificate[] x509Certificates, String s) {}
```

Comment:
Can we throw exception for client verfification `throw new UnsupportedOperationException("checkClientTrusted should not be called");` like we do in setupInsecureSsl

Reply:
Fixed. Thank you.